content/en/bits_ai/guide/query_examples.md

```diff
@@ -0,0 +1,75 @@
+---
```

Comment:
Would it be possible to move this out of Guides? This is one of our core features and given we have so few, can we start by pulling it out into its own page (at the same level as Managing Incidents)? The title of the page can be Natural Language Querying